CHANGELOG.md

```diff
@@ -4,6 +4,55 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-rc.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-rc.0) - 2022-10-12
+
+This is the first Zebra release candidate. Zebra's consensus rules, node sync, and `lightwalletd` RPCs are ready for final testing and experimental use. Zebra has not been audited yet.
```

Comment:
Is this accurate?
I think we wrote down how we wanted to explain the release, but I'm not sure where to find it.

Reply:
```suggestion
This is the first Zebra release candidate. Zebra's consensus rules, node sync, and `lightwalletd` RPCs are ready for user testing and experimental use. Zebra has not been audited yet.
```

Reply:
I'm not going to block the release on this, we can edit the exact wording later if we want to.